### PR TITLE
fix: typos & add typos-cli support

### PR DESCRIPTION
**Pull request by ben1009**

### CODE REVIEW

File: .typos.toml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# If there are words that you think should not be treated as typo.
+# Please list here along with a comment.
+[default.extend-words]
+"MUTLI" = "MUTLI"
+"SPLITTED" = "SPLITTED"
+"ba" = "ba"
+"hel" = "hel"
```
Comment (sagiegurari) [inline .typos.toml:7]: are these needed?

**Comment by sagiegurari** [comment]
@ben1009 didn't know of typos cli. its pretty cool.
got a question on the config file there....

File: .typos.toml (hunk 1 above)
Comment (ben1009) [reply .typos.toml:7]: same words in tests seems intend, not a typo, so just ignore these ones, make the check happy

**Comment by sagiegurari** [comment]
@ben1009 merged. thanks a lot

**Pull request merged**